<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientsFormView</class>
 <widget class="QWidget" name="clientsFormView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="3">
    <widget class="QPushButton" name="buttonAddYur">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="15" column="3" rowspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="2">
    <widget class="QLineEdit" name="editSearchYur"/>
   </item>
   <item row="12" column="3">
    <widget class="QPushButton" name="buttonDelYur">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="buttonSearchFiz">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QPushButton" name="buttonDocsYur">
     <property name="text">
      <string>К документам</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="buttonDelFiz">
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QPushButton" name="buttonSearchYur">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QPushButton" name="buttonEditYur">
     <property name="text">
      <string>Редактировать</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Клиенты юридические лица</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonAddFiz">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QPushButton" name="buttonAllYur">
     <property name="text">
      <string>Показать всех</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="buttonCasesFiz">
     <property name="text">
      <string>К делам</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="buttonDocsFiz">
     <property name="text">
      <string>К документам</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="buttonAllFiz">
     <property name="text">
      <string>Показать всех</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="buttonEditFiz">
     <property name="text">
      <string>Редактировать</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" rowspan="6" colspan="3">
    <widget class="QTableView" name="yurView"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="editSearchFiz"/>
   </item>
   <item row="0" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Клиенты физические лица </string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QPushButton" name="buttonCasesYur">
     <property name="text">
      <string>К делам</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="6" colspan="3">
    <widget class="QTableView" name="fizView"/>
   </item>
   <item row="6" column="3" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>186</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
